src/cljs/athens/electron/db_menu/db_list_item.cljs

```diff
@@ -59,10 +61,21 @@
 
 (defn db-list-item
   [{:keys [db is-current]}]
-  [:div (use-style db-list-item-style)
-   (if is-current
-     [:div.body.is-current
-      [db-list-item-content {:db db}]]
-     [:button.body.button {:onClick
-                           #(dispatch [:db-picker/select-db db])}
-      [db-list-item-content {:db db}]])])
+  (let [state                   (r/atom {:show-remove-button false})
```

Comment:
If you really have to have local state - and please don't have it, make it part of reframe -
you don't need to have a map with just 1 key, simpler would be to have `show? (r/atom false)`.